google-cloud-trace-v2/README.md

```diff
@@ -0,0 +1,75 @@
+# Ruby Client for the Cloud Trace V2 API
+
+API Client library for the Cloud Trace V2 API
+
+The Cloud Trace API lets you send and retrieve latency data to and from Cloud Trace. This API provides low-level interfaces for interacting directly with the feature. For some languages, you can use OpenCensus, a set of open source tracing and stats instrumentation libraries that work with multiple backends.
+
+https://github.com/googleapis/google-cloud-ruby
+
+## Installation
+
+```
+$ gem install google-cloud-trace-v2
+```
+
+## Before You Begin
+
+In order to use this library, you first need to go through the following steps:
+
+1. [Select or create a Cloud Platform project.](https://console.cloud.google.com/project)
+1. [Enable billing for your project.](https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project)
+1. [Enable the API.](https://console.cloud.google.com/apis/library/cloudtrace.googleapis.com)
+1. {file:AUTHENTICATION.md Set up authentication.}
+
+## Quick Start
+
+```ruby
+require "google/cloud/trace/v2"
+
+client = ::Google::Cloud::Trace::V2::TraceService::Client.new
+request = my_create_request
```

Comment:
Would it be possible to show something more realistic or detailed here in place of `my_create_request`?

Reply:
I guess it's possible, but not easy. The generator simply chooses the first method in the list to show in the quick start, and has little context about the method's semantics, and what a plausible request would look like. I think we're going to try to move in this direction, figuring out how to generate reasonable sample snippets without requiring huge configs. But the current text is my way of punting for now.